database/seeds/CustomerSeeder.php

```diff
@@ -33,8 +33,6 @@ public function run() {
                             ($faker->boolean(50) ? $faker->phoneNumber : null)
                     ]),
                     'email' => 'FFE-COMPANY-' . $i . '@mailinator.com',
-                    'parent_id' => ($faker->boolean(50) && ($i > 1)) ?
```

Comment:
Since we keep the parent_id field (as mentioned in my comment above), this can be kept in.